frontend/src/app/shared/components/autocompleter/project-autocompleter/project-autocompleter-template.component.html

```diff
@@ -4,7 +4,16 @@
   let-clear="clear"
   >
   <span class="ng-value-icon left" (click)="clear(item)">×</span>
-  <span class="ng-value-label">{{ item.name }}</span>
+  <span class="ng-value-label" style="display: flex; gap: 0.5rem;">
```

Comment:
We can use utility classes here: [`.d-flex` from Primer](`https://primer.style/product/css-utilities/flexbox/`) and our own (`.gap-*`)[src/global_styles/openproject/_gap.scss} introduced by @toy 

```suggestion
  <span class="ng-value-label" class="d-flex gap-3">
```